backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2924,4 +2924,24 @@ databaseChangeLog:
                   type: boolean
                   remarks: patient_has_prior_tests will track whether a patient has prior test events
                   constraints:
-                    nullable: true
\ No newline at end of file
+                    nullable: true
+  - changeSet:
+      id: add-indexes-top_level_dashboard_metrics
+      author: brett.m.maden@omb.eop.gov
+      changes:
+        - createIndex:
+            tableName: test_event
+            indexName: ix__test_event__prior_corrected_test_event_id
+            columns:
+              - column:
+                  name: prior_corrected_test_event_id
+        - createIndex:
+            tableName: test_event
+            indexName: ix__test_event__fac_id-coal_test_date-test-corr_status
+            columns:
+              - column:
+                  name: facility_id
+              - column:
+                  name: COALESCE(date_tested_backdate, created_at)
+              - column:
```

Comment:
Would it be possible to include the UUID of the person tested here as well? Other parts of the dashboard call for a per-facility count of the number of unique people tested, so we may want to include that in this index.

Reply:
It could be added as a 4th column on this index and still allow this query to use the index.  It would only be usable on the other query if it uses all 4 of the columns, though.

I haven't seen the other query, but if you can send me a link to it I'll check to see if we can add an index that can be used by both.

Reply:
I haven't written the other query yet I'm afraid :/ but it will essentially need to gather the following data for each facility:
- # total tests
- # unique people tested
- # positive tests
- # negative tests

Reply:
Hmmmm, it sounds like the query for getting the facility data for test counts (# 1, 3, 4) could use a query that makes use of this index, but I think it might take another query to get the number of unique people tested.  It shouldn't even need the join since the join to deduplicate the corrected tests since I think a distinct `patient_id` would take care of that already.

I don't _think_ it seems like this index could be reused by the other query, but if I'm wrong I'd be happy to modify it.

Reply:
Frankly I'm glad to hear you say that because I spent much of Friday trying to modify the original query to include unique people and couldn't figure out how to do it 😆 So using a separate query for that piece of data sounds like a good plan to me, thanks. Sorry to take you down this rabbit hole, this index LGTM as is!